src/main/java/org/apache/fineract/api/OperationsDetailedApi.java

```diff
@@ -227,74 +227,98 @@ public Page<TransactionRequest> transactionRequests(
      * @param page the count/number of page which we want to fetch
      * @param size the size of the single page defaults to [10000]
      * @param sortedOrder the order of sorting [ASC] or [DESC], defaults to [DESC]
-     * @param filterBy type of filter we want to apply, @see [Filter]
-     * @param ids the list of ids to match the query with
      */
     @PostMapping("/transactionRequests/export")
```

Comment:
Please follow REST principles for APIs. ResourceName/ResourceId/SubResourceName/SubResourceId?command=verbWhichWrites&verbQueryParamForRead=value

Reply:
@avikganguly01 what changes do you want me to do here?

Reply:
/transactionRequests?command=export